From: Marcel Pennewiss <gentoo@pennewiss.de>
Date: Thu Apr 12 12:20:15 CEST 2012
Subject: Fixes compilation

HomerSounds is not included in git repository
(based on #d16df92655)

--- HomerBuild/CMakeLists.txt
+++ HomerBuild/CMakeLists.txt
@@ -14,7 +14,6 @@
 ADD_SUBDIRECTORY(../HomerConference ${CMAKE_CURRENT_BINARY_DIR}/HomerConference)
 ADD_SUBDIRECTORY(../HomerSoundOutput ${CMAKE_CURRENT_BINARY_DIR}/HomerSoundOutput)
 ADD_SUBDIRECTORY(../Homer ${CMAKE_CURRENT_BINARY_DIR}/Homer)
-ADD_SUBDIRECTORY(../../Homer-Release/HomerSounds ${CMAKE_CURRENT_BINARY_DIR}/HomerSounds)
 
 ##############################################################################
 # include CPack configuration to generate installers for Win/OSX
